frappe/contacts/report/addresses_and_contacts/addresses_and_contacts.json

```diff
@@ -1,30 +1,33 @@
 {
- "add_total_row": 0, 
- "apply_user_permissions": 1, 
- "creation": "2017-01-19 12:57:22.881566", 
- "disabled": 0, 
- "docstatus": 0, 
- "doctype": "Report", 
- "idx": 2, 
- "is_standard": "Yes", 
- "modified": "2017-04-10 15:04:12.498920", 
- "modified_by": "Administrator", 
- "module": "Contacts", 
- "name": "Addresses And Contacts", 
- "owner": "Administrator", 
- "ref_doctype": "Address", 
- "report_name": "Addresses And Contacts", 
- "report_type": "Script Report", 
+ "add_total_row": 0,
+ "columns": [],
+ "creation": "2022-01-19 01:08:57.725304",
```

Comment:
```suggestion
 "creation": "2017-01-19 12:57:22.881566",
```

This has been broken for a while and I'll work on a fix soon. For the time being, keep an eye out for the `creation` key 🥲 

I've only undone this but there are more places this has changed in this PR